# DP: use gcc-4.9 instead of gcc as the command name.

Index: b/src/gcc/ada/comperr.adb
===================================================================
--- a/src/gcc/ada/comperr.adb
+++ b/src/gcc/ada/comperr.adb
@@ -367,7 +367,7 @@ package body Comperr is
                End_Line;
 
                Write_Str
-                 ("| Include the exact gcc or gnatmake command " &
+                 ("| Include the exact gcc-4.9 or gnatmake command " &
                   "that you entered.");
                End_Line;
 
Index: b/src/gcc/ada/gnatlink.adb
===================================================================
--- a/src/gcc/ada/gnatlink.adb
+++ b/src/gcc/ada/gnatlink.adb
@@ -137,7 +137,7 @@ procedure Gnatlink is
    --  This table collects the arguments to be passed to compile the binder
    --  generated file.
 
-   Gcc : String_Access := Program_Name ("gcc", "gnatlink");
+   Gcc : String_Access := Program_Name ("gcc-4.9", "gnatlink");
 
    Read_Mode : constant String := "r" & ASCII.NUL;
 
@@ -1475,7 +1475,8 @@ procedure Gnatlink is
       end if;
 
       Write_Line ("  --GCC=comp Use comp as the compiler");
-      Write_Line ("  --LINK=nam Use 'nam' for the linking rather than 'gcc'");
+      Write_Line ("  --LINK=nam Use 'nam' for the linking rather " &
+                  "than 'gcc-4.9'");
       Write_Eol;
       Write_Line ("  [non-Ada-objects]  list of non Ada object files");
       Write_Line ("  [linker-options]   other options for the linker");
Index: b/src/gcc/ada/make.adb
===================================================================
--- a/src/gcc/ada/make.adb
+++ b/src/gcc/ada/make.adb
@@ -670,7 +670,7 @@ package body Make is
    -- Compiler, Binder & Linker Data and Subprograms --
    ----------------------------------------------------
 
-   Gcc          : String_Access := Program_Name ("gcc", "gnatmake");
+   Gcc          : String_Access := Program_Name ("gcc-4.9", "gnatmake");
    Original_Gcc : constant String_Access := Gcc;
    --  Original_Gcc is used to check if Gcc has been modified by a switch
    --  --GCC=, so that for VM platforms, it is not modified again, as it can
Index: b/src/gcc/ada/gnatchop.adb
===================================================================
--- a/src/gcc/ada/gnatchop.adb
+++ b/src/gcc/ada/gnatchop.adb
@@ -45,7 +45,7 @@ procedure Gnatchop is
    Config_File_Name : constant String_Access := new String'("gnat.adc");
    --  The name of the file holding the GNAT configuration pragmas
 
-   Gcc : String_Access := new String'("gcc");
+   Gcc : String_Access := new String'("gcc-4.9");
    --  May be modified by switch --GCC=
 
    Gcc_Set : Boolean := False;
Index: b/src/gcc/ada/mdll-utl.adb
===================================================================
--- a/src/gcc/ada/mdll-utl.adb
+++ b/src/gcc/ada/mdll-utl.adb
@@ -39,7 +39,7 @@ package body MDLL.Utl is
    Dlltool_Name  : constant String := "dlltool";
    Dlltool_Exec  : OS_Lib.String_Access;
 
-   Gcc_Name      : constant String := "gcc";
+   Gcc_Name      : constant String := "gcc-4.9";
    Gcc_Exec      : OS_Lib.String_Access;
 
    Gnatbind_Name : constant String := "gnatbind";
@@ -212,7 +212,7 @@ package body MDLL.Utl is
          end;
       end if;
 
-      Print_Command ("gcc", Arguments (1 .. A));
+      Print_Command ("gcc-4.9", Arguments (1 .. A));
 
       OS_Lib.Spawn (Gcc_Exec.all, Arguments (1 .. A), Success);
 
Index: b/src/gcc/ada/mlib-utl.adb
===================================================================
--- a/src/gcc/ada/mlib-utl.adb
+++ b/src/gcc/ada/mlib-utl.adb
@@ -444,7 +444,7 @@ package body MLib.Utl is
       if Driver_Name = No_Name then
          if Gcc_Exec = null then
             if Gcc_Name = null then
-               Gcc_Name := Osint.Program_Name ("gcc", "gnatmake");
+               Gcc_Name := Osint.Program_Name ("gcc-4.9", "gnatmake");
             end if;
 
             Gcc_Exec := Locate_Exec_On_Path (Gcc_Name.all);
Index: b/src/gcc/ada/prj-makr.adb
===================================================================
--- a/src/gcc/ada/prj-makr.adb
+++ b/src/gcc/ada/prj-makr.adb
@@ -116,7 +116,7 @@ package body Prj.Makr is
 
    procedure Dup2 (Old_Fd, New_Fd : File_Descriptor);
 
-   Gcc      : constant String := "gcc";
+   Gcc      : constant String := "gcc-4.9";
    Gcc_Path : String_Access := null;
 
    Non_Empty_Node : constant Project_Node_Id := 1;
